content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.md

```diff
@@ -70,9 +70,7 @@ AccountRoot objects can have the following flag values:
 
 | Flag Name                         | Hex Value    | Decimal Value | Corresponding [AccountSet Flag](accountset.html#accountset-flags) | Description |
 |-----------------------------------|--------------|-------------------|-----------------------------------|----|
- `lsfAllowTrustLineClawback`        | `0x80000000` | 2147483648        |
-`asfAllowTrustLineClawback`         | Enable [Clawback](clawing-back-tokens.html) for this account. |
-| `lsfDefaultRipple`                | `0x00800000` | 8388608           | `asfDefaultRipple`                | Enable [rippling](rippling.html) on this addresses's trust lines by default. Required for issuing addresses; discouraged for others. |
+`lsfAllowTrustLineClawback`        | `0x80000000` | 2147483648        | `asfAllowTrustLineClawback`         | Enable [Clawback](clawing-back-tokens.html) for this account. | `lsfDefaultRipple`                | `0x00800000` | 8388608           | `asfDefaultRipple`                | Enable [rippling](rippling.html) on this addresses's trust lines by default. Required for issuing addresses; discouraged for others. |
```

Comment:
You appear to have combined one too many lines here (lsfDefaultRipple should stay on its own)